packages/ng-schematics/package.json

```diff
@@ -22,31 +21,23 @@
       "output": [
         "lib/**/files/**",
         "lib/**/*.json"
-      ],
-      "dependencies": [
-        "clean"
       ]
     },
     "build": {
       "command": "tsc -b",
       "files": [
```

Comment:
This need to be kept as is. Actually we should do do this for all our packages. If a step does not control part of the output dir we should excluded that part from output. That way wire it can deal properly changes. As at the moment is deleting the output form `build:copy` or some times not creating anything in `lib`.